coordinatorlib/src/main/scala/com/socrata/datacoordinator/common/soql/sqlreps/RepUtils.scala

```diff
@@ -38,4 +38,8 @@ abstract class RepUtils {
     }
     sb.append(q)
   }
+
+  def cleanText(s: String): String =
+    if(s.indexOf('\u0000') == -1) s // fastpath: no NULs to remove
```

Comment:
Why 4 0s? Isn't '\0' `nul`?

Reply:
Nope (used to be, but Scala got rid of that, which only really makes sense for C).  Have to use the generalized unicode escape sequence.